components/admin/requests/forms/new-applications/GuardianInformationForm.tsx

```diff
@@ -0,0 +1,203 @@
+import {
+  FormControl,
+  FormLabel,
+  Input,
+  Text,
+  Stack,
+  FormHelperText,
+  Box,
+  Divider,
+  Button,
+} from '@chakra-ui/react'; // Chakra UI
+import { AttachmentIcon } from '@chakra-ui/icons';
+import { GuardianInformation } from '@tools/components/admin/requests/forms/types'; // Guardian Information Type
+
+type GuardianInformationFormProps = {
+  readonly guardianInformation: GuardianInformation;
+  readonly onChange: (updatedData: GuardianInformation) => void;
+  readonly onFileUpload: (args: Record<string, any>) => void;
```

Comment:
In `GuardianInformation`, we can add a field for the files with type `FileList`. We shouldn't need an additional prop for `onFileUpload`